packages/client/src/Loc.ts

```diff
@@ -1298,8 +1306,9 @@ export class OpenLoc extends EditableRequest {
     }
 
     async publishFile(parameters: { hash: Hash } & BlockchainSubmissionParams): Promise<OpenLoc> {
+        // TODO: Check signer is requester
```

Comment:
To be addressed before release?

Reply:
Addressed by this issue: https://github.com/logion-network/logion-internal/issues/976